lib/concurrent-ruby/concurrent/executor/java_thread_pool_executor.rb

```diff
@@ -73,6 +73,11 @@ def completed_task_count
         @executor.getCompletedTaskCount
       end
 
+      # @!macro thread_pool_executor_attr_reader_available_worker_count
+      def available_worker_count
```

Comment:
As discussed, should this just be `available_count` or is the `worker_count` taxonomy established here/elsewhere?